src/metabase/driver/postgres.clj

```diff
@@ -770,3 +773,49 @@
   ;; This could be incorrect if Postgres has been compiled with a value for NAMEDATALEN other than the default (64), but
   ;; that seems unlikely and there's not an easy way to find out.
   63)
+
+(defn- format-copy
+  [_clause table]
+  [(str "COPY " (sql/format-entity table))])
+
+(sql/register-clause! ::copy format-copy :insert-into)
```

Comment:
I think it's a little weird we use namespaced keywords, but we did it before with `::union` so I'm just following the pattern. I guess it guarantees we'll never clash with a potential Honey SQL version